plugins/main/windows/win-overlay/win-overlay_windows.go

```diff
@@ -153,7 +163,8 @@ func cmdDel(args *skel.CmdArgs) error {
 
 	epName := hns.ConstructEndpointName(args.ContainerID, args.Netns, n.Name)
 
-	return hns.DeprovisionEndpoint(epName, args.Netns, args.ContainerID)
+	err = hns.DeprovisionEndpoint(epName, args.Netns, args.ContainerID)
```

Comment:
this looks odd; error doesn't seem to get used anywhere? I'm probably missing something.

Reply:
eg, return the error, or ignore it (with `_ = hns.DeprovisionEndpoint(...)` and `return nil`)